### DISCUSSION TITLE
Matplotlib and CustomTkinter Compatibility

### DISCUSSION THREAD

**Discussion by wilam24** [TomSchimansky/CustomTkinter#2207] [Q&A] [answered]
I want to ask about the compatibility of CustomTkinter and Matplotlib library. I have used Tkinter library for creating seminar presentation and to show some graph I've made using Matplotlib in the same GUI, and now I want do the same thing with CustomTkinter. 

1. Does CustomTkinter do the same thing as Tkinter did when handling a Matplotlib graphic?
2. Are there any issue that people had found about the compatibility of this library (CustomTkinter) with Matplotlib?
3. Is there any downside of using CustomTkinter (something that can be done in Tkinter but not in CustomTkinter)?

Thanks in advance!

**Comment by Akascape** [comment, marked as answer]
@wilam24 customtkinter is build in top of tkinter, so there should no issue implementing other libraries in it. If tkinter can support it, customtkinter should support it too. I don't think there should be any issue implementing matplotlib.
